### PR TITLE
Delete code related to NATIVETRAN feature.

### PR DESCRIPTION
**Pull request by waywardmonkeys**
This was obsolete work that had been done for generating
native code from the bytecode.

Closes Interlisp/medley#89.

### CODE REVIEW

**Comment by waywardmonkeys** [comment]
Trying to clear out some of the ifdefery in the dispatch and related code ... slimming it down is making it easier to understand.

File: src/testtool.c [hunk 1]
```diff
@@ -1196,7 +1196,6 @@ void all_stack_dump(DLword start, DLword end, DLword silent)
           print(get_framename((struct frameex1 *)stkptr));
           printf(" [");
           if (((FX *)stkptr)->fast) printf("fast,");
-          if (((FX *)stkptr)->native) printf("native,");
```
Comment (nbriggs) [inline src/testtool.c:1199]: This one should probably stay until we remove the "native" flag bit from the Frame Extension, likewise for the Function Header struct which also has a "native" flag.

Comment (waywardmonkeys) [reply src/testtool.c:1199]: I removed them in `inc/stack.h` and marked what they used to be. Or are you talking about Lisp side? But C side, everything is gone and a few foot prints left as a trail to the past.

Comment (nbriggs) [reply src/testtool.c:1199]: Looks as though I got started in the middle somewhere, so I'd missed that you'd dummied out the "native" flag bit.
If all the traces (bar history pointers) are gone, then this is fine.  I/we do need to look at the Lisp side to make sure there are no dangling references into the emulator code.

Comment (nbriggs) [reply src/testtool.c:1199]: Looks as though I got started in the middle somewhere, so I'd missed that you'd dummied out the "native" flag bit.
If all the traces (bar history pointers) are gone, then this is fine.  I/we do need to look at the Lisp side to make sure there are no dangling references into the emulator code.

**Comment by nbriggs** [comment]
This will take me a little while to review -- I'll look at it in the morning so I have uninterrupted time.

**Review by nbriggs — COMMENTED** [review]
LGTM (Remove unused SaveD6)

**Comment by waywardmonkeys** [comment]
Anything pending on this?

**Comment by nbriggs** [comment]
Not yet.  I'll be on it soon though.

**Review by nbriggs — APPROVED** [review]
LGTM.

**Pull request merged**